server/core/services.py

```
@@ -0,0 +1,87 @@
+
+        Raises:
+            ValidationError: If login fails
+        """
+        try:
+            # Log the user in (validates with HTTP session storage)
+            login(request, user)
+
+            return {
+                'user': user.to_dict()
+            }
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)